cmd/bootstrap/README.md

```diff
@@ -212,11 +282,29 @@ spec:
   bindHost: 127.0.0.1
 ```
 
-Now that the `west` site has been created, copy the generated token into
+##### Bootstrap the west site
+
+Considering all your CRs have been saved to a directory named `west`, use:
+
+```shell
+./bootstrap -p ./west
+```
+
+The CRs are defined using the `west` namespace, so it will be used by default.
+If the namespace was empty, skupper would try to set it to `default`.
+
+Now that the `west` site has been created, copy the generated link into
 the `east` site local directory. Example:
 
 ```shell
-cp ${HOME}/.local/share/skupper/sites/west/runtime/token/link-go-west.yaml ./east/links/link-go-west.yaml
+cp ${HOME}/.local/share/skupper/namespaces/west/runtime/link/link-go-west.yaml ./east/link-go-west.yaml
+```
+
+Make sure to update the namespace on the generated `link` that will be used
+in the `east` site. To do that, run:
+
+```shell
+sed -i 's/namespace: west/namespace: east/g' ./east/link-go-west.yaml
```

Comment:
When one does `skupper token issue` for a kube site we end up with a AccessToken resource without a namespace set - so that it ends up ready to apply to another site. Would it make sense to do the same for the generated accesstokens for non-kube sites?

Reply:
To make it easier as well, I have also modified the example defined at the README file to have all CRs
defined without a namespace. This way it will be easier to copy the static token from a site to another.